source/features/clean-conversation-sidebar.tsx

```diff
@@ -40,9 +40,14 @@ function cleanSection(selector: string): boolean {
 
 	const heading = select(':scope > details, :scope > .discussion-sidebar-heading', container)!;
 
-	// Magic. Do not touch.
-	// Section is empty if: no sibling element OR empty sibling element
-	if (heading.nextElementSibling?.firstElementChild) {
+	const labelsUniqueSelector = '.IssueLabel';
```

Comment:
Can you place them in a single array and then describe them with one-word comment?

And then use querySelector(arr.join())

Reply:
ok will do